addons/mail/views/mail_activity_views.xml

```diff
@@ -0,0 +1,146 @@
+<?xml version="1.0"?>
+<odoo>
+    <record id="mail_activity_view_form" model="ir.ui.view">
+        <field name="name">mail.activity.form</field>
+        <field name="model">mail.activity</field>
+        <field name="arch" type="xml">
+            <form string="Activities">
+                <div class="oe_title">
+                    <label for="name" class="oe_edit_only"/>
+                    <h1><field name="name"/></h1>
```

Comment:
`h1` is a little big extreme. I know most of people are wearing glasses, but the input is to big ;)